components/CookieSettings.tsx

```
@@ -0,0 +1,347 @@
+
+      {/* Quick Actions */}
+      <div className="mb-8 p-6 bg-gray-50 dark:bg-gray-800/50 rounded-lg">
+        <h2 className="text-lg font-semibold text-gray-900 dark:text-white mb-4">
+          {t('quickActions')}
+        </h2>
+        <div className="flex flex-wrap gap-3">
+          <button
+            onClick={handleAcceptAll}
+            className="px-4 py-2 text-sm font-medium text-white bg-primary-600 hover:bg-primary-700 dark:bg-primary-500 dark:hover:bg-primary-600 rounded-lg transition-colors"
+          >
+            {t('acceptAll')}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)